wifite/util/crack.py

```diff
@@ -13,14 +13,14 @@
 from ..tools.hashcat import Hashcat, HcxPcapTool
 from ..tools.john import John
 
-from datetime import datetime
+from json import loads
 
 import os
 
 
 # TODO: Bring back the 'print' option, for easy copy/pasting. Just one-liners people can paste into terminal.
 
-# TODO: Do not show handshake files that are in cracked.txt with a key (match on filename).
+# TODO: Add an ability to select multiple dictionaries from a directory (the directory may possibly be an argument to the script --wordlist-dir).
```

Comment:
That's a good idea.

Detecting wordlists would be easy using [`os.path.isdir(...)`](https://docs.python.org/3/library/os.path.html#os.path.isdir)... but I would have to check if all cracking tools support multiple wordlists as input.

Can you create an issue (Feature Request) and delete the `TODO` line? I completely forgot about the previous `TODO` and will probably forget this one too :disappointed: